diff --git a/etc/raved b/etc/raved
index 1d9dc34..177dc3c 100755
--- a/etc/raved
+++ b/etc/raved
@@ -41,7 +41,7 @@ get_rave_pid() {
 
 # Checks if the process is running
 check_rave_status() {
-  RAVEPGFPROCESS=`su $BALTRAD_USER -s /bin/bash -c "/opt/baltrad/rave/bin/rave_pgf status"`
+  RAVEPGFPROCESS=`su $BALTRAD_USER -s /bin/bash -c "/usr/lib/rave/bin/rave_pgf status"`
   if [ "$RAVEPGFPROCESS" = "rave_pgf is not running" ]; then
     return 1
   else
@@ -67,7 +67,7 @@ case "$1" in
     if [ $? -eq 0 ]; then
       echo "already running"
     else
-      su $BALTRAD_USER -s /bin/bash -c "/opt/baltrad/rave/bin/rave_pgf start"
+      su $BALTRAD_USER -s /bin/bash -c "/usr/lib/rave/bin/rave_pgf start"
       #/opt/baltrad/rave/bin/rave_pgf start
       if [ $? -ne 0 ]; then
         echo "failed."
@@ -78,7 +78,7 @@ case "$1" in
     ;;
   stop)
     echo -n "Stopping Rave..."
-    su $BALTRAD_USER -s /bin/bash -c "/opt/baltrad/rave/bin/rave_pgf stop"
+    su $BALTRAD_USER -s /bin/bash -c "/usr/lib/rave/bin/rave_pgf stop"
       if [ $? -ne 0 ]; then
         echo "failed."
       else
