--- a/etc/odiminjectord.service	2019-02-07 22:31:03.948514551 +0100
+++ b/etc/odiminjectord.service	2019-02-07 22:30:53.192760476 +0100
@@ -0,0 +1,14 @@
+[Unit]
+Description=ODIM injector daemon
+After=syslog.target network.target
+
+[Service]
+Type=forking
+PIDFile=/var/run/odim_injector/odim_injector.pid
+ExecStart=/usr/lib/rave/bin/odim_injector --pidfile=/var/run/odim_injector/odim_injector.pid --logfile=/var/log/baltrad/odim_injector.log --indir=/var/lib/baltrad/odim_injector/data
+ExecStop=/usr/lib/rave/bin/odim_injector --pidfile=/var/run/odim_injector/odim_injector.pid --kill
+User=baltrad
+Group=baltrad
+RuntimeDirectory=odim_injector
+RuntimeDirectoryPreserve=yes
+
+[Install]
+WantedBy=multi-user.target
