diff --git a/etc/raved b/etc/raved
index 38cc0c7..69277f0 100755
--- a/etc/raved
+++ b/etc/raved
@@ -15,9 +15,6 @@
 ### END INIT INFO
 
 PATH=/bin:/usr/bin:/sbin:/usr/sbin
-NAME=raved
-DESC="Baltrad rave-pgf server"
-DEFAULT=/etc/default/$NAME
 BALTRAD_VAR_DIR=/var/run/baltrad
 RAVEPID=${BALTRAD_VAR_DIR}/rave_pgf_server.pid
 
@@ -44,7 +41,7 @@ get_rave_pid() {
 
 # Checks if the process is running
 check_rave_status() {
-  RAVEPGFPROCESS=`/opt/baltrad/rave/bin/rave_pgf status`
+  RAVEPGFPROCESS=`/usr/lib/rave/bin/rave_pgf status`
   if [ "$RAVEPGFPROCESS"="rave_pgf is not running" ]; then
     return 1
   else
@@ -70,7 +67,7 @@ case "$1" in
     if [ $? -eq 0 ]; then
       echo "already running"
     else
-      su $BALTRAD_USER -s /bin/bash -c "/opt/baltrad/rave/bin/rave_pgf start"
+      su $BALTRAD_USER -s /bin/bash -c "/usr/lib/rave/bin/rave_pgf start"
       #/opt/baltrad/rave/bin/rave_pgf start
       if [ $? -ne 0 ]; then
         echo "failed."
@@ -81,7 +78,7 @@ case "$1" in
     ;;
   stop)
     echo -n "Stopping Rave..."
-    su $BALTRAD_USER -s /bin/bash -c "/opt/baltrad/rave/bin/rave_pgf stop"
+    su $BALTRAD_USER -s /bin/bash -c "/usr/lib/rave/bin/rave_pgf stop"
       if [ $? -ne 0 ]; then
         echo "failed."
       else
