--- baltrad-node.service	2019-02-07 22:31:03.948514551 +0100
+++ baltrad-node.service	2019-02-07 22:30:53.192760476 +0100
@@ -0,0 +1,23 @@
+[Unit]
+Description=Manages the tomcat servlet engine used by baltrad node.
+After=syslog.target network.target
+[Service]
+Type=forking
+Environment=CATALINA_HOME=/usr/share/baltrad/baltrad-node-tomcat
+Environment=CATALINA_BASE=/var/lib/baltrad/baltrad-node-tomcat
+Environment="JAVA_OPTS=-Djava.awt.headless=true -Dfile.encoding=UTF-8 -server -Xms1536m -Xmx1536m -XX:NewSize=256m -XX:MaxNewSize=256m -XX:+DisableExplicitGC"
+Environment=CATALINA_PID=/var/run/baltrad/baltrad-node-tomcat.pid
+Environment=CATALINA_TMPDIR=
+Environment=JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk
+Environment=LD_LIBRARY_PATH=/usr/lib/java:$LD_LIBRARY_PATH
+Environment=UMASK=0002
+PIDFile=/var/run/baltrad/baltrad-node-tomcat.pid
+ExecStart=/usr/share/baltrad/baltrad-node-tomcat/bin/catalina.sh start
+ExecStop=/usr/share/baltrad/baltrad-node-tomcat/bin/catalina.sh stop
+User=baltrad
+Group=baltrad
+RuntimeDirectory=baltrad
+RuntimeDirectoryPreserve=yes
+
+[Install]
+WantedBy=multi-user.target
