--- a/etc/baltrad-exchange.service	2019-02-07 22:31:03.948514551 +0100
+++ b/etc/baltrad-exchange.service	2019-02-07 22:30:53.192760476 +0100
@@ -0,0 +1,17 @@
+[Unit]
+Description=Baltrad exchange server
+After=syslog.target network-online.target
+Wants=network-online.target
+
+[Service]
+Type=forking
+PIDFile=/var/run/baltrad/baltrad-exchange-server.pid
+ExecStart=/usr/bin/baltrad-exchange-server --pidfile=/var/run/baltrad/baltrad-exchange-server.pid --conf=/etc/baltrad/baltrad-exchange.properties --logfile=/var/log/baltrad/baltrad-exchange-server.log
+KillSignal=SIGHUP
+User=baltrad
+Group=baltrad
+RuntimeDirectory=baltrad
+RuntimeDirectoryPreserve=yes
+
+[Install]
+WantedBy=multi-user.target
