From 1f269a0a80d8c580a01616ac21b9036ecf26238a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20DAVID?= <davidcl@fedoraproject.org>
Date: Wed, 25 Jan 2012 18:51:05 +0100
Subject: [PATCH 4/7] remove writable prefix check

---
 configure.in |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/configure.in b/configure.in
index c1b4b06..c4a2fcb 100755
--- a/configure.in
+++ b/configure.in
@@ -644,10 +644,6 @@ if test $JH45INST_FOUND -eq 1; then
   if test \( ! -d "$JH45INST" \); then
     JH45INST_FOUND=0
   fi
-  if test \( ! -w "$JH45INST" \); then
-    AC_MSG_ERROR( [ $JAVINST : not writable ])
-    JH45INST_FOUND=0
-  fi
 fi
 until test $JH45INST_FOUND -eq 1; do
   echo "Please type the directory in which to install the JH45"
@@ -656,10 +652,6 @@ until test $JH45INST_FOUND -eq 1; do
   if test \( ! -d "$JH45INST" \); then
     JH45INST_FOUND=0
   fi
-  if test \( ! -w "$JH45INST" \); then
-    AC_MSG_ERROR( [ $JAVINST : not writable ])
-    JH45INST_FOUND=0
-  fi
 done
 
 ac_cv_lib_JH45INST=$JH45INST
-- 
1.7.7.6

